pages/admin/request/create-new.tsx

```diff
@@ -245,10 +245,14 @@ export default function CreateNew() {
     },
   });
 
+  // TODO: move to a different file
```

Comment:
Is this comment still needed?

Reply:
Yeah that hasn't been addressed yet, I was thinking it would be better to move the schema to a file in `tools` or `lib`, but I just originally put it there so it would be easier to work with while implementing all the forms. We could also just leave it where it currently is if you're okay with that?